cuebot/src/main/java/com/imageworks/spcue/rqd/RqdClientGrpc.java

```diff
@@ -42,23 +50,41 @@
 
     private final int rqdServerPort;
 
+    private final int rqdCacheSize = 500;
+    private final int rqdCacheExpiration = 30;
```

Comment:
Should these be static / camelCase?

Reply:
Moved these so they are read in from the properties file.